src/index.html

```diff
@@ -234,30 +234,60 @@ <h2>SKILLS</h2>
       </section>
 
       <!-- Information -->
-      <div>
-        <section>
-          <h2>QUICK ID</h2>
-          <ul>
-            <li><b>Phone</b><a href="tel:123-456-7890">123-456-7890</a></li>
-            <li>
-              <b>Email</b><a href="mailto:info@mysite.com">info@mysite.com</a>
-            </li>
-            <li>
-              <b>Website</b
-              ><a href="http://www.wix.com" target="_blank">www.wix.com</a>
-            </li>
-            <li>
-              <b>Address</b>
-              <address>
-                500 Terry Francois Street San Francisco, CA 94158
-              </address>
-            </li>
-          </ul>
-        </section>
+      <div id="banner_information">
+        <section id="banner_information_quick_id">
+          <h2 class="banner_title">QUICK ID</h2>
+          <table id="banner_information_quick_id_tb">
+            <tr>
+              <td class="quick_id_tb_label"><b>Phone</b></td>
+              <td class="quick_id_tb_content">
+                <a href="tel:123-456-7890">123-456-7890</a>
+              </td>
+            </tr>
+
+            <tr></tr>
```

Comment:
empty tr without td is against the Semitic html. Rmove the empty tr and just use either 
- padding of th/td
- cellpadding